analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd

```diff
@@ -320,6 +326,28 @@ neurocytoma_results_df <- neurocytoma_results_df %>%
          Notes = if_else(!is.na(integrated_diagnosis), "Updated via OpenPBTA subtyping", Notes))
 ```
 
+### Handling Chordoma
+```{r}
+
+chordoma_results_df <- bind_exp_strategies(chordoma_results_df) %>%
+  inner_join(select(histologies_df,
+                    Kids_First_Biospecimen_ID,
+                    short_histology,
+                    broad_histology,
+                    Notes),
+             by = "Kids_First_Biospecimen_ID") %>%
+  # will update subtype and integrated diagnosis in step 3
+  mutate(molecular_subtype = NA_character_,
+         integrated_diagnosis = NA_character_,
+         broad_histology = "Choroid plexus tumor",
```

Comment:
```suggestion
         broad_histology = "Chordoma",
```
actually, this is wrong in our current file, so we should change it here - I have a note to update this in v19, but I can add it to the v19 ticket as well.

I also think lines 329-350 here can be removed and broad and short histology added within script 03, similar to the rest of the samples requiring pathology review. Were you adding it here in order to add "Updated via PBTA subtyping?"

Reply:
Do we want to remove all lines 329-350? Or did you mean I should keep the broad_histology,short_histology and Notes NA till we update the values from pathology review?

I added the chordoma subtyping here (like we do for all other subtyping results ) since 01 is the script that actually compiles all the subtyping info and in 03 we update the values from pathology review.

Reply:
Sorry for the delay - I went back through this and think as you have it now is good. Thanks!